scripts/minishift-execute-tests.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+shout() {
+   set +x
+  echo -e "\n.---------------------------------------\n${1}\n'---------------------------------------\n"
+   set -x
+}
+
+set -ex
+
+eval $(minishift oc-env)
+
+shout "| Logging in to minishift..."
+oc login -u developer -p developer --insecure-skip-tls-verify $(minishift ip):8443
+
+shout "| Executing: generic, login, component command and plugin handler integration tests"
+make test-generic
```

Comment:
Can we consolidate test execution into one file together with https://github.com/openshift/odo/pull/4401/files and tests that are executed against our PSI OCP cluster and also Prow?

The list of tests is stored in multiple places, and it will be almost impossible to remember all the places where new test need to be added :-( Having it in one place would help a lot.

Reply:
Or we could simply create `make test-all` and use that everywhere

Reply:
Sure, I will consolidate in one script